pallets/msa/src/tests.rs

```diff
@@ -1220,45 +1207,21 @@ pub fn remove_msa_delegation_call_has_correct_costs() {
 }
 
 #[test]
-pub fn revoke_delegation_by_provider_errors_when_no_delegator_msa_id() {
```

Comment:
it still errors when the delegation deletion fails, but I removed the test case since the check is done in the SignedExtension.